src/Hosting/Systemd/src/ServiceState.cs

```diff
@@ -35,7 +35,7 @@ public ServiceState(string state)
         /// String representation of service state.
         /// </summary>
         public override string ToString()
-            => _data == null ? string.Empty : Encoding.UTF8.GetString(_data);
```

Comment:
Can still be null using the struct default constructor.

Reply:
Smart guy